testing/api/test_aggregate_slicewise.py

```diff
@@ -316,7 +316,7 @@ def test_save_as_csv_extract_metric(tmp_path, dummy_data_and_labels):
     with open(path_out, 'r') as csvfile:
         spamreader = csv.reader(csvfile, delimiter=',')
         next(spamreader)  # skip header
-        assert next(spamreader)[1:-1] == [__version__, '', '0:4', '', 'label_0', '2.5', '38.0']
+        assert next(spamreader)[1:-1] == [__version__, '', '0:4', '', '', 'label_0', '2.5', '38.0']
 
 
 def test_dimension_mismatch_between_metric_and_vertfile(dummy_metrics, dummy_vert_level):
```

Comment:
Can we add some tests that demonstrate typical usage of this feature? 

e.g. How would we expect the user to specify the `-pmj`, `-distance`, and `-extent` arguments? Then validating the expected output.

Reply:
> Can we add some tests that demonstrate typical usage of this feature?

yes I'll write up some tests!

Reply:
should we also add the usage to `batch_processing.sh`?

Reply:
I added a few tests in:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/cd420b4ca247115a37212f50eda794af0ed77476, https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/d151a92858f42fcbbf6acd40cc4f9b0807f6f70d, https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/0e32c430aed61ef4b54b80d9e0d511fd0b42e7e6

I want to add the method in `batch_processing.sh` with the related tests, but I can also address it in an other PR